### PR TITLE
Fix: removes a migration hack in the metrics app

### PR DESCRIPTION
**Pull request by lsh-0**
it eased the transition from renaming 'metrics' to 'article-metrics' and introducing a new 'metrics' table

cc @giorgiosironi

### CODE REVIEW

**Comment by lsh-0** [comment]
and now elife-metrics on 18.04 and python3.6 is returning different results to 14.04 and python3.4 ... :sigh:

**Comment by lsh-0** [comment]
destroyed my venv and I can recreate it under python 3.5. installing and testing on python 3.6 ...

**Comment by lsh-0** [comment]
two nasty problems: bad data at GA returning empty responses and tests calling GA. PR here: https://github.com/elifesciences/elife-metrics/pull/87

File: src/metrics/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,52 +1,17 @@
...
-    except BaseException:
-        LOG.error("unhandled exception attempting to update django_migrations.app values")
-        raise
-
-#
-#
-#
+# lsh 2019-03-27: delete once the flag is absent
+OLD_FLAG = Path(settings.PROJECT_DIR, ".migrations-table-updated.flag")
+OLD_FLAG.exists() and OLD_FLAG.unlink()
```
Comment (giorgiosironi) [inline src/metrics/migrations/0001_initial.py]: May be more clear with an `if`

Comment (lsh-0) [reply src/metrics/migrations/0001_initial.py]: just as clear. it will be gone on Friday regardless

**Comment by lsh-0** [comment]
fixing up issue in elife-metrics--ci

File: src/metrics/migrations/0001_initial.py (hunk 1 above)
Comment (giorgiosironi) [reply src/metrics/migrations/0001_initial.py]: I agree it doesn't really matter in this case (and approved), but in general what is clear/understandable will be up to the reader of this code :sweat_smile: it's like saying "I don't understand matrix calculus"/"it's just numbers"

Comment (lsh-0) [reply src/metrics/migrations/0001_initial.py]: again, I disagree. the above is perfectly fine. it's *not* like 

>  "I don't understand matrix calculus"/"it's just numbers"

it's more like 

> "I don't understand boolean logic", "but you're a programmer!!!"

**Comment by lsh-0** [comment]
dependent PR merged, giving this a kick

**Pull request merged**